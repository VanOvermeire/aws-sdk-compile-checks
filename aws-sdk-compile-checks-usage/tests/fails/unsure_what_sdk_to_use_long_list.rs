use aws_sdk_compile_checks_macro::required_props;
use aws_sdk_lambda::Client;

#[required_props]
async fn do_call(client: Client) {
    client.tag_resource()
        .send()
        .await
        .expect("Call to succeed");
}

fn main() {}
