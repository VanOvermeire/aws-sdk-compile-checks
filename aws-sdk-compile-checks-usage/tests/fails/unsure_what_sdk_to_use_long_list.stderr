error: method `tag_resource` is used in multiple SDKs: accessanalyzer, alexaforbusiness, amp, amplify, amplifyuibuilder... (abbreviated list). Please add the right one(s) to the attribute, e.g. `#[required_props(sdk = accessanalyzer)]`
 --> tests/fails/unsure_what_sdk_to_use_long_list.rs:6:12
  |
6 |     client.tag_resource()
  |            ^^^^^^^^^^^^
